Tutorial_5_BonusNotebook.ipynb

```diff
@@ -0,0 +1,756 @@
+   "metadata": {},
+   "source": [
+    "### STEP 1: Install Mambaforge and then install snakemake using bioconda."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First install Mambaforge.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "<YOUR COMMAND HERE>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "<details>\n",
+    "  <summary>Click for help</summary>\n",
+    "    \n",
+    "  Make sure you include the `!` in front of your command!  \n",
+    "    \n",
+    "\n",
+    "`curl -L -O https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-$(uname)-$(uname -m).sh` \n",
+    "`bash Mambaforge-$(uname)-$(uname -m).sh -b -u -p $HOME/mambaforge` \n",
+    "`date +\"%T\"` \n",
+    "\n",
+    "</details>\n",
+    "\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Next, add it to the path"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "<YOUR COMMAND HERE>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<details>\n",
+    "  <summary>Click for help</summary>\n",
+    "    \n",
+    "    \n",
+    "\n",
+    "`#add to your path`  \n",
+    "`import os`  \n",
+    "`os.environ[\"PATH\"] += os.pathsep + os.environ[\"HOME\"]+\"/mambaforge/bin\"`  \n",
+    "\n",
+    "\n",
+    "</details>\n",
+    "\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Next, using mambaforge and bioconda, install the tools that will be used in this tutorial."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "<YOUR COMMAND HERE>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "<details>\n",
+    "  <summary>Click for help</summary>\n",
+    "    \n",
+    "\n",
+    "`mamba install -y -c conda-forge -c bioconda trimmomatic fastqc multiqc salmon entrez-direct gffread parallel-fastq-dump sra-tools pigz`\n",
+    "\n",
+    "\n",
+    "</details>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### STEP 2: Setup Environment"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The directory structure will be the same as in Tutorial 1B. Create that below."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "<YOUR COMMAND HERE>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<details>\n",
+    "  <summary>Click for help</summary>\n",
+    "    \n",
+    "\n",
+    "` cd $HOMEDIR`  \n",
```

Comment:
maybe remove $HOMEDIR cause it will make a directory outside of the repo directory

Reply:
Good idea.